R/engine.R

```diff
@@ -658,6 +658,54 @@ eng_go = function(options) {
   engine_output(options, code, extra)
 }
 
+# SASS / SCSS engine
+## converts SASS / SCSS -> CSS (with same treatments as CSS engine) using either:
+## LibSass sass R package (https://github.com/rstudio/sass) if installed, or
+## dart-sass standalone executable (https://sass-lang.com/install)
+eng_sxss = function(options) {
+
+  if (!options$eval) {
+    return(engine_output(options, options$code, ''))
+  }
+
+  # create temporary file with input code
+  f = tempfile( pattern = 'code', tmpdir = '.', fileext = paste0(".",options$engine) )
```

Comment:
I think it is best to follow the style of the rest of the code here
```suggestion
  f = tempfile(pattern = 'code', tmpdir = '.', fileext = paste0(".", options$engine))
```

Reply:
Makes sense - sorry! I'll clean this up here and other similar places throughout the function.